src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -36,6 +37,7 @@
     const GROUPS = 'groups';
     const ATTRIBUTES = 'attributes';
     const ALLOW_EXTRA_ATTRIBUTES = 'allow_extra_attributes';
+    const EMPTY_DATA = 'default_constructor_arguments';
```

Comment:
Might as well rename the `EMPTY_DATA` constant then :)

Reply:
Arg!